onboard-julia/add-user.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {useState} from 'react';
+import {Alert, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {styles} from './styles';
+import {
+  isValidBirthdate,
+  isValidEmail,
+  isValidName,
+  isValidPassword,
+  isValidPhone,
+  isValidRole,
+} from './user-validation';
+
+export function AddUser(): React.JSX.Element {
+  const [name, setName] = useState<string>('');
+  const [phone, setPhone] = useState<string>('');
+  const [birthdate, setBirthdate] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [role, setRole] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+
+  const handleAddUserPress = () => {
+    if (!isValidName(name)) {
+      Alert.alert('Erro', 'Por favor, insira um nome válido.');
+      return;
+    }
+
+    if (!isValidEmail(email)) {
+      Alert.alert('Erro', 'Por favor, insira um e-mail válido.');
+      return;
+    }
+
+    if (!isValidPhone(phone)) {
+      Alert.alert('Erro', 'Por favor, insira um telefone válido.');
+      return;
+    }
+
+    if (!isValidBirthdate(birthdate)) {
+      Alert.alert('Erro', 'Por favor, insira uma data de nascimento válida.');
+      return;
+    }
+
+    if (password.length < 7) {
+      Alert.alert('Erro', 'As senhas devem ter no mínimo 7 caracteres.');
+      return;
+    }
+
+    if (!isValidPassword(password)) {
+      Alert.alert(
+        'Erro',
+        'As senhas devem ter no mínimo uma letra e um número',
+      );
+      return;
+    }
+
+    if (!isValidRole(role)) {
+      Alert.alert('Erro', 'Por favor, insira um role válido.');
```

Comment:
aqui ta misturando ingles/portugues, acho que vale manter tudo de uma forma

```suggestion
      Alert.alert('Erro', 'Por favor, insira uma função válida.');
```